src/components/BorrowerProfile/CommentsAndWhySpecial.vue

```diff
@@ -278,13 +340,29 @@ export default {
 				this.isLightboxVisible = false;
 				this.loading = false;
 			});
-		}
+		},
+		randomizedUserClass() {
+			const randomStyle = this.userCardStyleOptions[Math.floor(Math.random() * this.userCardStyleOptions.length)];
+			return `${randomStyle.color} ${randomStyle.bg}`;
+		},
+		determineIfMobile() {
+			this.isMobile = document.documentElement.clientWidth < 735;
+		},
+		isDefaultProfilePic(commentHash) {
+			return isLegacyPlaceholderAvatar(commentHash);
+		},
+		throttledResize() {
+			return _throttle(this.determineIfMobile, 200);
+		},
 	},
 	mounted() {
 		this.createObserver();
+		window.addEventListener('resize', this.throttledResize());
```

Comment:
I think here you probably don't want the function call, since it will be returning a new instance of `_throttle`, so just `window.addEventListener('resize', this.throttledResize);`. Same with for remove.

Reply:
Whoops, absolutely right

Reply:
As a side note there is quite a bit of the incorrect setup with _throttle in the listener used throughout the UI app